Makefile.in

```diff
@@ -161,18 +161,28 @@ DEST_EMAN	= $(man1dir)/$(EMAN)
 #
 all: $(CTAGS_EXEC) $(READ_LIB) $(READ_CMD)
 
+#
+# verbose $(CC) compiler call definition
+#
+# when V is not set the output of $(CC) is simplified
+#
+V	?= 0
+V_CC	= $(V_CC_$(V))
+V_CC_0	= @echo [CC] $@; $(CC)
+V_CC_1	= $(CC)
```

Comment:
I'd rather have `V_CC` only expand to empty or an `@echo` command -- e.g. not including `$(CC)`.
Yes, this will mean slight duplication, but it is more robust (e.g. would still work if the user sets `V=2`) and allows to have preliminary commands hidden by it, like your `mkdir` below.